test/e2e/deployment.go

```diff
@@ -458,7 +458,7 @@ func testRolloverDeployment(f *framework.Framework) {
 	// Create a deployment to delete nginx pods and instead bring up redis-slave pods.
 	// We use a nonexistent image here, so that we make sure it won't finish
 	deploymentName, deploymentImageName := "test-rollover-deployment", "redis-slave"
-	deploymentReplicas := int32(4)
+	deploymentReplicas := int32(3)
```

Comment:
Can't do this, since rollover test requires at least 3 replicas: it creates an RS first, then create a deployment that creates another RS, and then before the rolling update finishes, update deployment to create the third RS to rollover the first two versions